### PR TITLE
fix linting run

### PR DESCRIPTION
**Pull request by janfjohannes**
---
[//]: # (BEGIN SAPLING FOOTER)
* #3052
* __->__ #3051

### CODE REVIEW

File: lix/packages/fs/src/utilities/helpers.ts [hunk 1]
```diff
@@ -18,19 +18,25 @@ export function assertIsAbsolutePath(path: string) {
...
 }
 
-export function normalizePath(path: string, { trailingSlash, leadingSlash }: { trailingSlash?: 'always' | 'strip', leadingSlash?: 'always' } = {}): string {
-	path = path.replace(/^\.\//, '/')
+export function normalizePath(
+	path: string,
+	{
+		trailingSlash,
+		leadingSlash,
+	}: { trailingSlash?: "always" | "strip"; leadingSlash?: "always" } = {}
```
Comment (samuelstroschein) [inline lix/packages/fs/src/utilities/helpers.ts:26]: Shall we agree on a convention with leading and trailing slashes? Making it configurable seems to be a mistake that needs to be reconsiled with tons of normalization functions. 

If it's a directory, it feels natural to enforce a trailing slash to have a distinction to files. 

I did not find a spec for OPFS https://fs.spec.whatwg.org/#locator-path. Seems like everyone has a different (or no?) spec for path.

File: lix/packages/fs/src/utilities/helpers.test.ts [hunk 2]
```diff
@@ -17,11 +17,11 @@ test("not add a slash", () => {
...
-	expect(normalizePath("project.inlang", { leadingSlash: 'always'})).toBe("/project.inlang")
+	expect(normalizePath("project.inlang", { leadingSlash: "always" })).toBe("/project.inlang")
 })
 
 test("should add trailing slash", () => {
-	expect(normalizePath("project.inlang", { trailingSlash: 'always' })).toBe("project.inlang/")
+	expect(normalizePath("project.inlang", { trailingSlash: "always" })).toBe("project.inlang/")
 })
 
 describe("keep heading slashes free for windows norm", () => {
```
Comment (samuelstroschein) [inline lix/packages/fs/src/utilities/helpers.test.ts:27]: Why do we care about windows? 

Given that we control the lix environment e2e, I would spec our path implementation (as mentioned in the other comment)

**Review by samuelstroschein — APPROVED** [review]
Are you running prettier before you push? I had several whitespace/formatting changes. Seems like we are running a different prettier version. 

EDIT: I see no prettier config exists in the lix SDK package.

File: lix/packages/fs/src/utilities/helpers.ts (hunk 1 above)
Comment (samuelstroschein) [reply lix/packages/fs/src/utilities/helpers.ts:26]: https://github.com/opral/lix-sdk/issues/30

File: lix/packages/fs/src/utilities/helpers.test.ts (hunk 2 above)
Comment (samuelstroschein) [reply lix/packages/fs/src/utilities/helpers.test.ts:27]: https://github.com/opral/lix-sdk/issues/30

File: lix/packages/fs/src/utilities/helpers.ts (hunk 1 above)
Comment (janfjohannes) [reply lix/packages/fs/src/utilities/helpers.ts:26]: @samuelstroschein  this are only automated changes from prettier, i did not touch anything manually that did not exist before

Comment (samuelstroschein) [reply lix/packages/fs/src/utilities/helpers.ts:26]: oh lol this is not even in the SDK package ^^

**Pull request merged**